lib/rbac/filterer.rb

```diff
@@ -451,6 +448,11 @@ def method_with_scope(ar_scope, options)
     end
 
     def apply_scope(klass, scope)
+      klass = to_class(klass)
```

Comment:
`to_class` takes a string and calls constantize on it. For a class it is a no-op.

I want just this call, but wasn't able to delete line 177 yet.

It will be a no-op.